### ISSUE TITLE
Explain how `TextSpan::locale` influences voice on  'Accessibility' page

### ISSUE THREAD

**Issue by Leffe108** [flutter/website#6878]
## Page URL

https://docs.flutter.dev/development/accessibility-and-localization/accessibility/

## Page Source

https://github.com/flutter/website/tree/main/src/development/accessibility-and-localization/accessibility.md

## Describe the problem

In order to markup which voice locale that assistive technology like TalkBack and VoiceOver should use, a Flutter user/developer has to use `TextSpan::locale`. This is however neither mentioned in [accessibility doc](https://docs.flutter.dev/development/accessibility-and-localization/accessibility) nor [internationalization doc](https://docs.flutter.dev/development/accessibility-and-localization/internationalization)

It could also help to clear up that the locale property on MaterialApp and Localizations.override does not affect which voice assistive technologies like VoiceOver and TalkBack is using as opposed to how the lang attribute works in HTML. [Reference](https://github.com/flutter/flutter/issues/99018#issuecomment-1057497573)

## Expected Fix

At the end of the [Screen reader section](https://docs.flutter.dev/development/accessibility-and-localization/accessibility#screen-readers), add a paragraph

> When there is text is your app that should be voiced with a specific voice, use [`TextSpan.locale`](https://api.flutter.dev/flutter/painting/TextSpan/locale.html) to inform the screen reader about the voice to use. Note that `MaterialApp.locale` and `Localizations.override` do not affect which voice is used by the screen reader. Usually the screen reader will use the system voice to voice texts except where you explicitly told it with `TextSpan.locale`.
